pkg/location/location.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package location
+
+import (
+	"fmt"
+)
+
+type Location struct {
```

Comment:
Thinking from an end-user API's perspective:

```

loc := content.Location{}

src, err := content.Open(loc)

```

Reply:
TBH - the struct Location was only added as a convenience when a method _returned_ both values - in places like content.Open when a ref or refLock is an _argument_ you'd be interested in one or the other, but both at once never really happened. Closest would be like upgrade - but even then the "@version" mutates the Reference and two calls to content.Open was more natural...

In fact - even when Location is used to return both values - there's always other values being returned as well. So even there it's not necessary.

In hindsight I think it'd be best to remove the `content.Location{}` 2-value struct because like you say it _looks_ like something you should use to open content, but isn't